recsys.egg-info/requires.txt

```diff
@@ -0,0 +1,9 @@
+joblib==0.13.2
```

Comment:
Suggest to remove the package such as python-dateutil, joblib , pytz that are not used.